### ISSUE TITLE
Multi-Data Center Support in Akka.Cluster

### ISSUE THREAD

**Issue by Aaronontheweb** [akkadotnet/akka.net#3261]
This is something that's been in the works on the JVM for about 6-7 months and we've had some requests for it over the past couple of years on the Akka.NET project. I'm interested in making this happen as part of the Akka.NET v1.4 or v1.5 milestones.

**Comment by feitzi** [comment]
is there any target date for data-center support in akka.net (maybe during 2019?)

**Comment by Aaronontheweb** [comment]
We dropped it for the 1.4.0 release coming up because no one other than @Horusiath (the author) has had the bandwidth to review the code / finish verifying that the multi-node specifications work and we're not in a position to do that immediately, but we should be able to do it this year.

**Comment by feitzi** [comment]
thanks aaron, this year sounds good 👍  
Can we support you in this topic?

**Comment by Horusiath** [comment]
This PR is pretty much done - it's working and tests on .net full framework are passing. The issue is in .NET Core multi-node tests, which are failing for some reason, but since MNTK runner is not working locally for me I'm unable to find the issue.

**Comment by tcs-devel** [comment]
Do you have any plans for a 1.5 release, or any release including this feature?
Thanks!

**Comment by tcs-devel** [comment]
Any plans when 1.5.0 / 2.0.0 with this feature should be available?

**Comment by tcs-devel** [comment]
Just checking if there are any further plans for this?
Thank you!

**Comment by Aaronontheweb** [comment]
@tcs-devel yep, we do plan on implementing this - I'm not sure if it's in-scope for v1.6 or not at this rate simply because there hasn't been a huge amount of demand for it and it is a fairly significant, complicated-to-test feature.

I personally think it's a good idea for disaster recovery scenarios though, so I would like to see it added and evangelized. I think it's one of those things where demand for it will increase once it's available as a viable option.
